src/functions/asyncApi2PayloadValidation.ts

```diff
@@ -0,0 +1,60 @@
+import * as AJV from 'ajv';
```

Comment:
Whoa, I'd rather not do this, since it'll inline the whole ajv, significantly increasing the size of a function.